### PR TITLE
#756: [calypso] Added simctl repopulate

### PR DESCRIPTION
**Pull request by bonkey**
#756: [calypso] Added simctl repopulate

### CODE REVIEW

**Comment by bonkey** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

**Comment by bonkey** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

**Comment by bonkey** [comment]
👍

File: lib/calypso/simctl.rb [hunk 1]
```diff
@@ -151,13 +151,18 @@ def repopulate_all_simulators(name = nil)
...
-      simulators = available_runtimes
-      if !name.nil?
-        simulators = simulators.select { |runtime| runtime['name'] == name }
-      end
 
-      simulators.each do |runtime|
+      selected_runtimes = unless name.nil?
+                            requested_runtimes = available_runtimes.select { |runtime| runtime['name'] == name }
+                            if requested_runtimes.length == 0
+                              log "No simulators created, \"#{name}\" not found"
```
Comment (bonkey) [inline lib/calypso/simctl.rb]: Maybe `"No runtimes for #{name} found?"`

Comment (bonkey) [inline lib/calypso/simctl.rb]: And maybe `log_abort "..." if requested_runtimes.empty?`

**Comment by hani-ibrahim** [comment]
👍

**Comment by bonkey** [comment]
👍

**Pull request merged**